app/main/index.js

```diff
@@ -5,6 +5,7 @@ const windowStateKeeper = require('electron-window-state');
 const isDev = require('electron-is-dev');
 const appMenu = require('./menu');
 const { appUpdater } = require('./autoupdater');
+const { updateDebianNotification } = require('./debianupdater');
```

Comment:
You need to move this inside the `if` statement in line 167, otherwise the process will crash since this is a main process.

Reply:
@cPhost I don't think that would be enough.

Reply:
It is working fine now though without crashing. Why would it crash?

Reply:
@akashnimare i think that enough since semver is used in `debianupdater.js` and checking is done in this file. Does that make sense?

@abhigyank we remove the semver package in platform is not linux since you require a modules that not present it will throw an error and therefore crash the app.

(This is getting messy here, I'd rather prefer discussing stuff on chat.zulip.org)